v3/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/go-ldap/ldap/v3
+module github.com/go-ldap/ldap
```

Comment:
👀 You accidently deleted the v3 module definition

Reply:
It's a good thing someone is paying attention